src/auralis/models/xttsv2/utils/checkpoint_converter.py

```diff
@@ -24,14 +24,16 @@ def download_repo_files(repo_id, output_path, exclude_extensions=['*.safetensors
 
 
 
-def convert_checkpoint(pytorch_checkpoint_path, output_dir, args):
```

Comment:
In this file you should create a config.json file, be aware that in the download repo function you should be aware that it is downloading the config.json form out hf repo, and this overrides the config.json file you  should create